dbt_platform_helper/providers/validation.py

```diff
@@ -0,0 +1,16 @@
+class ValidationException(Exception):
```

Comment:
Does this want to inherit from PlatformException?

Reply:
Done.